fetch.bs

```diff
@@ -1505,6 +1566,28 @@ is a <a>filtered response</a> whose
  <li><p>Return <var>newResponse</var>.
 </ol>
 
+<p>To create a <dfn for=response id=concept-response-sanitized-copy>sanitized copy</dfn> of a
+<a for=/>response</a> <var>response</var>, run these steps:
+
+<ol>
+ <li><p>If <var>response</var> is a <a>network error</a>, then return a copy of <var>response</var>.
+
+ <li><p>Let <var>unfilteredResponse</var> be <var>response</var>'s <a>internal response</a> if
+ <var>response</var> is a <a>filtered response</a>, otherwise <var>response</var>.
+
+ <li><p>Let <var>sanitizedResponse</var> a new <a for=/>response</a> whose
+ <a for=response>status</a> is <var>unfilteredResponse</var>'s <a for=response>status</a>,
+ <a for=response>HTTPS state</a> is <var>unfilteredResponse</var>'s <a for=response>HTTPS state</a>,
+ and <a for=response>CSP list</a> is <var>unfilteredResponse</var>'s <a for=response>CSP list</a>.
+
+ <li><p>If <var>response</var> is a <a>filtered response</a>, set <var>sanitizedResponse</var> to
+ a new identical filtered response whose <a>internal response</a> is <var>sanitizedResponse</var>.
```

Comment:
"identical filtered response" feels a little hand-wavey, but it's what "clone" does above.